src/Chamber.sol

```diff
@@ -106,6 +106,30 @@ contract Chamber is IChamber, Common {
 
         if (!onVoterList) revert invalidApproval("TokenId not on voter list");
 
+        if ((proposals[_proposalId].approvals+1)==3){
+            // Gas estimation for the first proposal
+            uint256 baseGas = 110539;
+
+            // Average gas difference for each additional proposal
+            uint256 averageGasDifference = 31560;
```

Comment:
Where did you get this value from, and how can you be sure this is valid?

Reply:
Please read this 
https://github.com/loreum-org/chamber/pull/47#issuecomment-1905242615